lib/blockstore/index.js

```diff
@@ -0,0 +1,42 @@
+/*!
+ * blockstore/index.js - bitcoin blockstore for bcoin
+ * Copyright (c) 2019, Braydon Fuller (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const {join} = require('path');
+
+const AbstractBlockStore = require('./abstract');
+const LevelBlockStore = require('./level');
+const FileBlockStore = require('./file');
+
+/**
+ * @module blockstore
+ */
+
+exports.create = (options) => {
+  const location = join(options.prefix, 'blocks');
+
```

Comment:
Do you think its a good idea to do some type checking here?

Reply:
There are assertions on the options within the stores. I don't think it's necessary.